numpy/core/tests/test_multiarray.py

```diff
@@ -4868,7 +4867,7 @@ def test_field_rename(self):
 
     def test_multiple_field_name_occurrence(self):
         def test_assign():
-            dtype = np.dtype([("A", "f8"), ("B", "f8"), ("A", "f8")])
+            np.dtype([("A", "f8"), ("B", "f8"), ("A", "f8")])
```

Comment:
Test name should be changed here any below to not include the word `assign`, since it no longer does an assignment